### ISSUE TITLE
[FEATURE] Request for classic TAH layout

### ISSUE THREAD

**Issue by weepingminotaur** [Larkinabout/fvtt-token-action-hud-core#99]
I appreciate the recent changes to TAH, in particular the ability to list trained skills in blue and the increased customization. I definitely think it’s more developed that I’m considering switching over from the old version. However, I do want to offer a comparison of the old compact and new compact modes (see screenshot), in order to make a plea for the new TAH compact style to look like the old one.

In the old TAH, each action or Strike was neatly listed in a right-justified column on the left, which allowed attack info and draw/sheathe/drop to appear smoothly across the rest of the line. Minimal spacing ensured that everything was snugly laid out.

In the new TAH, that information is spread out across multiple lines even in compact. The indentations don’t really present the same clean, functional look as the old. There’s just … too much information spilling over onto multiple lines, even when I choose to hide a subcategory title (for example, Strikes).

In short, I would suggest making action titles appear on the same line as draw-sheathe-drop and the MAP attacks, as the old TAH did. Or basically having a “display using classic TAH” option in Core.
<img width="741" alt="Screen Shot 2023-04-09 at 2 05 02 PM" src="https://user-images.githubusercontent.com/90105219/230789159-c6e2876d-578b-4e92-ad81-81167bdf896b.png">

**Comment by Larkinabout** [comment]
I'll be honest I don't have a solution for this yet, which is why it hasn't happened. I tried tackling subtitles on the left as a starter, but haven't managed to make it work as yet. There's good reasons why I moved away from it (apart from liking it better this way myself), as the structure of the HUD is inherently different from the old HUD in order to support nested subcategories and customisability, and it was already prone to causing problems in the original TAH due to it's limited width and pushing out to the side, which cause problems with positioning.

With the customisability of TAH Core, you can however move all MAP attacks, Damage and Critical buttons onto the same line as the Draw and Sheathe buttons right now in the current release if you wish. I may eventually add a module setting to allow the user to choose to do that by default to save the hassle of moving it for every new weapon.

I'm going to link this issue to the other one asking for the original Strikes layout and close this one to avoid duplication. It's not off the table, but I don't have a quick fix, so it's likely to stay as is until I get some inspiration or someone else solves it.
